_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,279 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements have child nodes allowed by the content model
+accessibility_requirements:
+  wcag20:1.3.1: # Info and Relationships (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - Accessibility tree
+  - CSS Styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Wilco Fiers
+  previous_authors:
+    - Jey Nandakumar
+---
+
+## Applicability
+
+This rule applies to any [HTML elements][] with a [hidden state][] of "false", where the [semantic role][] is the same as the [implicit role][], and for which one of the following is true:
+
+- **lists**: The element is a `ul`, `ol` or `menu`; or
+- **definition lists**: The element is a `dl`, or a `div` that is a [child][] in the [flat tree][] of a `dl`.
```

Comment:
I might be mistaken, but I don't think a `div` can have an implicit role (I couldn't find anywhere that states that it is has one even as a child of a `dl`), so it cannot ever be applicable.

Reply:
You're right. I've redefined this in terms of an explicit role. Think this addressed the issue.